commit 8d2423c584f75dec236d957f3130b270f1197f79 (refs/changes/98/4498/1)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-07-09T19:28:18-04:00 (1 year, 3 months ago)
    
    joyent/node-verror#55 package.json repository does not point to Joyent

diff --git a/package.json b/package.json
index 79295c5..763a64f 100644
--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
 	"main": "./lib/verror.js",
 	"repository": {
 		"type": "git",
-		"url": "git://github.com/davepacheco/node-verror.git"
+		"url": "git://github.com/joyent/node-verror.git"
 	},
 	"dependencies": {
 		"assert-plus": "^1.0.0",
